verticapy/machine_learning/model_selection/statistical_tests/ols.py

```diff
@@ -58,6 +58,91 @@ def het_breuschpagan(
     tuple
         Lagrange Multiplier statistic, LM pvalue,
         F statistic, F pvalue
+
+    Examples
+    ---------
+
+    Let's try this test on a dummy dataset that has the
+    following elements:
+
+    - X (a predictor)
+    - True value
+    - Prediction
+
+    .. note::
+
+        This metric requires 'eps', which represents
+        the difference between the predicted value
+        and the true value. If you already have 'eps'
+        available, you can directly use it instead of
+        recomputing it, as demonstrated in the example
+        below.
+
+    Before we begin we can import the necessary libraries:
+
+    .. ipython:: python
+
+        import verticapy as vp
+
+    Now we can create the dummy dataset:
+
+    .. ipython:: python
+
+        vdf = vp.vDataFrame(
+            {
+                "X": [0,1,2,3],
+                "Y_true": [1.1, 1.2, 1.3, 1.4],
+                "Y_pred": [1.15, 1.23, 1.26, 1.4],
+            }
+        )
```

Comment:
is it possible to use a dataset with more elements?
Also is it possible to compute a linear regression out of it and show the plot of the linear regression.
You can then show the scatter plot of the residuals to show what they look like.